app/models/vm_or_template/scanning.rb

```diff
@@ -40,7 +40,7 @@ def scan(userid = "system", options = {})
     begin
       self.last_scan_attempt_on = Time.now.utc
       save
-      job = Job.create_job("VmScan", options)
+      job = ::Job.create_job("VmScan", options)
```

Comment:
Does this deserve a comment?  I know I won't remember why we're calling the toplevel Job here.